tests/acceptance/x-select-multiple-blockless-test.js

```diff
@@ -33,8 +33,9 @@ describe('XSelect: Multiple Selection', function() {
   });
 
   it('marks all selected values', function() {
-    expect(this.$('option:eq(1)')).to.be.selected;
-    expect(this.$('option:eq(2)')).to.be.selected;
+    expect(this.$('option:eq(0)').is('[selected]')).to.be.falsy;
+    expect(this.$('option:eq(1)').is('[selected]')).to.be.truthy;
+    expect(this.$('option:eq(2)').is('[selected]')).to.be.truthy;
```

Comment:
I'm not sure. My guess was that Glimmer wasn't able to set the actual DOM property, but only the attribute.